nominees/aws-infrastructure.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "AWS Infrastructure",
+  "description": "This project contains the AWS CloudFormation and Terraform templates to bootstrap AWS resources for Africa COVID-19 Response Toolkit projects.",
+  "website": "https://ethiocovid19rt.com/projects",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/africa-covid-19-response-toolkit/aws-infrastructure/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3,
+      "evidenceURL": "https://ethiocovid19rt.com/about"
```

Comment:
@rachita19082 in as much as possible, try to have evidenceURL point to external blogs, articles rather than owner's website pages. It would be good to make the above update for this before we can have it approved and merged. Thanks.